hadoop/src/main/java/org/apache/carbondata/hadoop/streaming/CarbonStreamRecordWriter.java

```diff
@@ -181,75 +181,81 @@ private void initializeAtFirstRow() throws IOException, InterruptedException {
       initializeAtFirstRow();
     }
 
-    // parse and convert row
-    currentRow.setData(rowParser.parseRow((Object[]) value));
-    converter.convert(currentRow);
-
     // null bit set
     nullBitSet.clear();
-    for (int i = 0; i < dataFields.length; i++) {
-      if (null == currentRow.getObject(i)) {
-        nullBitSet.set(i);
+    Object[] rowData = (Object[]) value;
+    currentRow.setRawData(rowData);
+    // parse and convert row
+    currentRow.setData(rowParser.parseRow(rowData));
+    CarbonRow updatedCarbonRow = converter.convert(currentRow);
+    if (updatedCarbonRow == null) {
+      output.skipRow();
+      currentRow.clearData();
+    } else {
```

Comment:
@geetikagupta16 Thanks for working on this
Hope no change in the else part, only formatting changed due the if check.

Reply:
@mohammadshahidkhan  Yes, else part remains the same